apps/backend/src/eventHandlers/MailService/SMTPMailService.ts

```diff
@@ -174,6 +174,22 @@ export class SMTPMailService extends MailService {
           id: 'isis-rapid-proposal-submitted-uo',
           name: 'ISIS Rapid User Office Submission Email',
         },
+        {
+          id: 'exp-proposal-submitted',
+          name: 'Express PI Co-I Submission Email',
+        },
+        {
+          id: 'exp-proposal-review',
+          name: 'Express update to Under Review Status Email',
+        },
+        {
+          id: 'exp-proposal-request-can',
+          name: 'Express update to Successful - request can Email',
```

Comment:
What is triggering this email and what is its purpose? I think these names could be clearer and follow the existing convention since they will be shown in the frontend. The existing convention is `Facility Recipient Event` e.g. `ISIS Xpress PI Co-I Approved Email`